bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -1135,6 +1136,34 @@ public ClientConfiguration setReorderReadSequenceEnabled(boolean enabled) {
         return this;
     }
 
+    /**
+     * If read operation should be sticky to a single bookie or not.
+     *
+     * @return true if reorder read sequence is enabled, otherwise false.
+     */
+    public boolean isStickyReadsEnabled() {
```

Comment:
can it be a per read request flag rather than per client?
i.e. we have various cases, in some cases reads are random and sticky reads probably won't help.
In other cases reads are strictly sequential and sticky read may help. "may" is because we can hit bottleneck on bookie NIC side and spreading reads across multiple bookies might actually help more then hurt.
Having two clients for these cases would mean 2x connections to bookies, executors/threads etc.

Reply:
you mean per ledger handle flag, not per read request flag, right?

Reply:
> can it be a per read request flag rather than per client?

@dlg99 It could work on per-request as long as the mechanism works across multiple requests. Eg: `read(0, 99)` and `read(100, 199)`, all the 200 requests should go to same bookie.

Reply:
> read(0, 99) and read(100, 199), all the 200 requests should go to same bookie.

won't happen for us, ES > WQ. but with https://github.com/apache/bookkeeper/pull/1808/files#r233679775 in mind that should not make any difference.
also we don't use read(0, 99) option. it happened so that we issue read(entryId) requests individually.